synapse/cmds/cortex.py

```diff
@@ -31,33 +44,16 @@ def runCmdOpts(self, opts):
             return
 
         core = self.getCmdItem()
-
         stormopts = {'repr': True}
-
         self.printf('')
 
         for mesg in core.storm(text, opts=stormopts):
 
             if opts.get('debug'):
                 self.printf(pprint.pformat(mesg))
-                continue
 
-            if mesg[0] == 'init':
-                continue
-
-            if mesg[0] == 'print':
-                self.printf(mesg[1].get('mesg'))
-                continue
-
-            if mesg[0] == 'warn':
-                warn = mesg[1].get('mesg')
-                self.printf(f'WARNING: {warn}')
-                continue
-
-            if mesg[0] == 'node':
+            elif mesg[0] == 'node':
```

Comment:
Is there a reason we can't use a Reactor here?

Reply:
We could use a Reactor here, but this PR was really to just add docs for all the args/flags for the cmd and not to rewrite it